src/test/java/com/timgroup/statsd/StatsDAggregatorTest.java

```diff
@@ -182,6 +162,7 @@ public void aggregate_messages() throws Exception {
 
         for(int i=0 ; i<10 ; i++) {
             fakeProcessor.send(new FakeMessage<Integer>("some.gauge", Message.Type.GAUGE, 1));
+            fakeProcessor.send(new FakeMessage<Integer>("some.time", Message.Type.TIME, 1));
```

Comment:
Hi @metao1 so, what I think Vikentiy meant was that we should make these into:
```
fakeProcessor.send(new FakeMessage<Integer>(new String("some.gauge"), Message.Type.GAUGE, 1));
```
because otherwise the string get's interned and if for whatever reason we have a regression in the aggregation logic (like going back to `==` from `equals()`) we'd be unable to catch the issue.

If you can make that update that would be awesome, otherwise we can follow-up on our end.

Reply:
nice that make sense to check with a new object to see if equal also works for interned string and string objects (which I pre-assumed it will). I always use equal() to check equality, and in very rare cases when using interned Strings will use ==.
 I added in test, please check new changes.